cmd/agent/main.go

```diff
@@ -485,3 +512,70 @@ func reconcileKubelet(ctx context.Context) (bool, error) {
 	}
 	return false, nil
 }
+
+func KubeletNeedsNewCertificate(headers http.Header) error {
+	ipAddress := os.Getenv("CATTLE_ADDRESS")
+	fileSafeIpAddress := strings.ReplaceAll(ipAddress, ".", "-")
+	certFileLocation := fmt.Sprintf("/etc/kubernetes/ssl/kube-kubelet-%s.pem", fileSafeIpAddress)
```

Comment:
I don't remember off the top of my head, but is there a way in RKE to define a different SSL directory? If so, can we use the logic from RKE to determine if the directory is what we expect? There's also a ton of helper logic for this here: https://github.com/rancher/rke/blob/b1afa9911a9111dffafde3c2e8eddce2fbcf1926/pki/util.go#L255

Reply:
I couldn't find an option in RKE which would allow users to change the location of the certs in the downstream node. There is a `--certs-dir` but that flag is for reading custom certs from the machine which is running `rke up`, and doesn't control the location they're added to in the downstream node